### PR TITLE
Fixes the syndie multitool tech origin, adds unit test for origin tech

### PR DESCRIPTION
**Pull request by Coolrune206**
## What Does This PR Do

Fixes a bug with the syndie multitool, making it yield proper tech levels again. So, fixes https://github.com/ParadiseSS13/Paradise/issues/19727
Also, adds a unit test for origin techs. Thanks AffectedArc.
## Why It's Good For The Game

Bugs bad. Unit tests good.
## Images of changes

image
## Testing

See above.
## Changelog
:cl:
fix: Fixed the Syndie Multitool tech origin
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Making a unit test for this would be good

```dm
var/regex/nums = regex("^[0-9]+")
for(var/tpath in subtypesof(/obj/item))
    var/obj/item/I = tpath
    var/tech_str = initial(I.origin_tech)

    var/list/tech_list = params2list(tech_str)

    for(var/k in tech_list)
    if(length(nums.Replace(tech_list[k], "")) > 0)
            Fail("Invalid origin tech for [tpath]: [tech_str]")
```

Theres the actual test run code, just integrate it into a `/datum/unit_test` in the way the others are.

**Comment by Coolrune206** [comment]
Trying to add the unit test to this PR causes it to fail CI and I'm unsure why, so I'll limit the scope of this PR to be a quick fix of the multitool issue.

**Comment by AffectedArc07** [comment]
> Trying to add the unit test to this PR causes it to fail CI and I'm unsure why, so I'll limit the scope of this PR to be a quick fix of the multitool issue. 

It likely fails CI due to another item having the same scuffed issue. Throw it in and ping me and I'll see what the fail reason is.

**Review by AffectedArc07 — APPROVED** [review]
CI change approved

File: code/modules/unit_tests/origin_tech.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/datum/unit_test/origin_tech/Run()
```
Comment (warriorstar-orion) [inline code/modules/unit_tests/origin_tech.dm:1]: Could you add a tiny docstring to this test?

**Review by farie82 — APPROVED** [review]

**Pull request merged**